.github/workflows/pr.yml

```diff
@@ -24,3 +24,30 @@ jobs:
       - name: Display structure of downloaded files
         run: ls -R
         working-directory: .
+  dependabot-comments:
+    runs-on: ubuntu-18.04
+    # Key change: modify the permission for this job taht ran by dependabot
+    # Doc: https://docs.github.com/en/actions/reference/authentication-in-a-workflow#modifying-the-permissions-for-the-github_token
+    permissions:
+      actions: read
+      checks: read
+      contents: read
+      deployments: read
+      issues: write
```

Comment:
Though we only wanted to modify the permission on issues as write, but we still need to specify the permission for all the others because of [doc](https://docs.github.com/en/actions/reference/authentication-in-a-workflow#modifying-the-permissions-for-the-github_token).

> When the permissions key is used, all unspecified permissions are set to no access, with the exception of the metadata scope, which always gets read access.